.github/workflows/core.yml

```diff
@@ -470,7 +470,7 @@ jobs:
       - run: chmod +x core/build/unix/trezor-emu-core*
       - uses: ./.github/actions/environment  # XXX poetry maybe not needed
       - run: nix-shell --run "git clone --depth=1 https://github.com/bitcoin-core/HWI.git"
-      - run: nix-shell --arg fullDeps true --run "cd HWI && poetry install && poetry run ./test/test_trezor.py --model_t ../core/build/unix/trezor-emu-core bitcoind"
+      - run: nix-shell --arg fullDeps true --run "cd HWI && poetry install && uv run ./test/test_trezor.py --model_t ../core/build/unix/trezor-emu-core bitcoind"
```

Comment:
Do we need both `poetry` and `uv` here (and in `legacy.yml`)?

Reply:
Right, `uv run` fails on my system with `error: No `project` table found in: /home/mmilata/work/HWI/pyproject.toml`. Doesn't really matter since the HWI jobs are disabled.

Reply:
yeah good catch, HWI is using poetry so that's what we stick with

Reply:
ad983e290776c5e8cf369b47464be611772ef750